ofcourse-templates/master.html

```diff
@@ -1,8 +1,14 @@
 <!DOCTYPE html>
+{% macro title() %}
+  {{course.name}}@{{course.place}}
+{% endmacro %}
+{% macro glyphicon(glyph) %}
+<span class="glyphicon glyphicon-{{glyph}}"></span>
```

Comment:
macros++

Does it make sense to register these outside of "master", or should they just live here?